swift/Vapor/README.md

```diff
@@ -0,0 +1,123 @@
+# The Algolia Client In Vapor
+
+## Futures
+
+The Algolia Swift client uses callbacks for handling the asynchronous calls.
```

Comment:
I tried those sync ones, but the calls were just hanging. There is no problem with the callbacks, the main idea of NIO is to be non-blocking everywhere. It's just easier to work with Futures. If there was a way to initialise the `SearchClient` with an `EventLoop`, this could all be hidden. But that is major work I guess? What can be done maybe is to provide these wrappers for everything out of the box—for the Linux target only.

Reply:
Great idea. I'll give it a try. Looks like a nice way to migrate our pseudo-async/await methods. But as you mentioned, it's a major work so it risks to be delayed in the backlog.